.github/workflows/code-check.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         uses: bluefireteam/melos-action@v3
 
       - name: Analyze packages
-        run: melos analyze
+        run: melos analyze --fatal-infos
```

Comment:
https://melos.invertase.dev/commands/analyze#--fatal-infos

Added because `melos analyze` results in SUCCESS even with warnings at the info level.